.storybook/config.js

```diff
@@ -6,28 +6,9 @@ import styled from 'styled-components';
 
 import '../src/global-styles';
 import { COLORS } from '../src/constants';
+import Wrapper from './components/Wrapper';
 
-const OuterWrapper = styled.div`

```

Comment:
It looks like the styled components was removed from this file so `import styled from 'styled-components';` should prob be removed as well?